prime-router/src/main/kotlin/tokens/SenderUtils.kt

```diff
@@ -0,0 +1,99 @@
+package gov.cdc.prime.router.tokens
```

Comment:
This is the beginnings of a set of tools that could potentially be given to Senders, to help them use FHIR-style Auth.    Although I'm guessing we might rewrite this in some other language.